lib/clearance/session.rb

```diff
@@ -62,8 +64,12 @@ def cookies
       @cookies ||= @env['action_dispatch.cookies'] || Rack::Request.new(@env).cookies
     end
 
+    def remember_token_cookie
```

Comment:
There are a few other methods later which are related to looking at the Clearance config. What do you think about moving this method down there?